manifests/2.0.0/opensearch-2.0.0-test.yml

```diff
@@ -65,7 +65,7 @@ components:
       test-configs:
         - without-security
 
-  - name: observability
+  - name: opensearch-observability
```

Comment:
The name changed? https://github.com/opensearch-project/opensearch-build/blob/main/manifests/1.3.1/opensearch-1.3.1-test.yml#L88

Reply:
The name was never right the whole time, thus obs test is skipped every single time.
```
2022-04-21 02:18:10 INFO     Skipping integ-tests for opensearch-observability, as it is currently not declared in the test manifest
```